src/components/cartdetail/PageCartDetail.vue

```diff
@@ -8,7 +8,11 @@
         <!--{{> checkout/start-checkout-link id="cart-checkoutnow-btn"}}-->
       </div>
     </div>
-    <div v-if="cartNotEmpty">
+    <div v-if="isLoading"
+         class="loading-spinner">
+      <img data-test="spinner" src="../../assets/img/spinner.gif"/>
+    </div>
+    <div v-else-if="!isLoading && cartNotEmpty">
```

Comment:
`isLoading=true` can never happen here, as this will already enter `v-if`. So `v-else-if` will always be `isLoading=false`.